components/QuizCardPage/User/index.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from "@testing-library/react";
+import { StyledUser1 } from ".";
+import { StyledUser2 } from ".";
```

Comment:
Both styled components are exactly the same, so you can use one component for both users.